app/Http/Controllers/InvoiceController.php

```diff
@@ -11,10 +11,40 @@ public function index()
     {
         $invoices = DB::table('invoices')
             ->join('customers', 'invoices.customer_id', '=', 'customers.id')
-            ->get();
-        // dd($invoices);
+            ->get([
+                'invoices.id AS id',
+                'invoice_date',
+                'first_name',
+                'last_name',
+                'total',
```

Comment:
There is a subtle bug in the last PR. The ID being displayed in the invoices table isn't the correct ID. This happened because of the join we did. Here we are fixing that by specifying what table the `id` value should come from.